### DISCUSSION TITLE
LOOPERS

### DISCUSSION THREAD

**Discussion by GateDrack** [skyline-emu/skyline#1524] [Game Requests]
Would be cool if you can add this, thanks

![hero](https://user-images.githubusercontent.com/107294554/173175934-ce6d36df-7f63-4090-af06-b833f2c319f9.jpeg)

**Comment by Crytonics** [comment]
I will test that game later.

**Reply by Crytonics** [reply]
Here: https://github.com/skyline-emu/skyline-games-list/issues/622
